assets/src/realtime.ts

```diff
@@ -53,26 +53,59 @@ export interface Ghost {
 
 export type NotificationId = string
 
-export interface Notification {
-  id: NotificationId
+// Matches the Module in which this notification was created by
+export enum NotificationType {
+  BridgeMovement = "Elixir.Notifications.Db.BridgeMovement",
+  BlockWaiver = "Elixir.Notifications.Db.BlockWaiver",
+}
+
+export interface BridgeLoweredNotification {
+  $type: NotificationType.BridgeMovement
+  status: "lowered"
+}
+
+export interface BridgeRaisedNotification {
+  $type: NotificationType.BridgeMovement
+  status: "raised"
+  loweringTime: Date
+}
+
+export type BridgeNotification =
+  | BridgeLoweredNotification
+  | BridgeRaisedNotification
+
+export type BlockWaiverNotification = {
```

Comment:
`createdAt` is part of the base type `NotificationContentTypes`, but it is also included in the `BlockWaiverNotification` type. If I'm understanding this right, I think the `createdAt` can be removed from this type spec?

Reply:
I'm not quite sure why they're duplicated, but I was trying to not introduce or remove any behavior in this PR.

You'll find that this column exists on both the notifications table and the block waiver table, and they seem to mean _something_ different between them? But I think that meaning is hidden deeper in the block waiver code.

https://github.com/mbta/skate/blob/310c7270adce01d8492e3baabf7c5d2601b57e83/lib/notifications/db/notification.ex#L13
https://github.com/mbta/skate/blob/310c7270adce01d8492e3baabf7c5d2601b57e83/lib/notifications/db/block_waiver.ex#L9

Reply:
Ohhhhh I see. How odd that it exists in both places, but I completely agree with your approach to avoid adding/subtracting existing behavior